addons/pos_self_order/controllers/self_entry.py

```diff
@@ -56,6 +56,7 @@ def start_self_ordering(self, config_id=None, access_token=None, table_identifie
                             'table': table._get_self_order_data() if table else False,
                             'access_token': config_access_token,
                             **pos_config._get_self_ordering_data(),
+                            **pos_config.get_kitchen_printer(),
```

Comment:
```suggestion
```
Instead of calling it here, call it inside `_get_self_ordering_data`. Put it in the key `kitchen_printers`.